### PR TITLE
feat: add liveness blog and update styling

### PR DESCRIPTION
**Pull request by 0xArdy**

### CODE REVIEW

File: src/data/blog.json [hunk 1]
```diff
@@ -1,4 +1,12 @@
 [
+  {
+    "id": "liveness-2-and-beyond",
+    "name": "Liveness 2 and beyond",
+    "description": "",
+    "date": "11/17/23",
+    "tags": [],
+    "image": "https://defi-wonderland.notion.site/image/https%3A%2F%2Fprod-files-secure.s3.us-west-2.amazonaws.com%2F7683bccd-1174-4689-a817-b27fd9d7ef00%2F8e9b1568-aea1-478f-af39-74c39502a79d%2FDALLE_2023-11-01_15.45.54_-_Create_an_epic_and_dark_illustration_featuring_Alice_from_Alice_in_Wonderland_actively_engaged_in_the_process_of_synchronizing_two_different_worlds._.png?table=block&id=79955f2a-d4eb-49c6-b7fe-8b729f841958&spaceId=7683bccd-1174-4689-a817-b27fd9d7ef00&width=2000&userId=&cache=v2"
```
Comment (turtlemoji) [inline src/data/blog.json:8]: should we use the image from notion or have it in another place hosted? @0xArdy @0xGorilla

Comment (0xGorilla) [reply src/data/blog.json:8]: Yes, this should be done on another task tho

File: src/data/blog.json [hunk 2]
```diff
@@ -1,4 +1,12 @@
 [
+  {
+    "id": "liveness-2-and-beyond",
+    "name": "Liveness 2 and beyond",
+    "description": "",
```
Comment (0xGorilla) [inline src/data/blog.json:5]: The current status of on-chain storage proofs

File: src/data/blog.json [hunk 3]
```diff
@@ -1,4 +1,12 @@
...
+    "date": "11/17/23",
+    "tags": [],
```
Comment (0xGorilla) [inline src/data/blog.json:7]: Cryptography, Verification, Cross-chain

File: public/blog-posts/liveness-2-and-beyond.md [hunk 4]
```diff
@@ -0,0 +1,318 @@
+# Liveness 2 and beyond
```
Comment (0xGorilla) [inline public/blog-posts/liveness-2-and-beyond.md:1]: Blog posts are not looking good on mobile

image

**Review by 0xGorilla — CHANGES_REQUESTED** [review]

File: src/data/blog.json (hunk 2 above)
Comment (0xArdy) [reply src/data/blog.json]: done https://github.com/defi-wonderland/web/pull/57/commits/f199666272a4c20f6846129c7ee6802e7116c65f

File: src/data/blog.json (hunk 3 above)
Comment (0xArdy) [reply src/data/blog.json]: done https://github.com/defi-wonderland/web/pull/57/commits/f199666272a4c20f6846129c7ee6802e7116c65f

**Comment by turtlemoji** [comment]
Card is still overflowing for me on blog
image

**Comment by 0xArdy** [comment]
> Card is still overflowing for me on blog image

fixed here https://github.com/defi-wonderland/web/pull/57/commits/62546cc61d15cd1dac23b8035683ea9fe18462d8

**Review by turtlemoji — APPROVED** [review]

**Review by 0xGorilla — APPROVED** [review]

**Pull request merged**